src/SFA.DAS.LevyTransferMatching.Web/Views/Opportunities/Apply.cshtml

```diff
@@ -110,11 +110,11 @@
                             <span class="das-task-list__task-tag">
                                 @if (Model.Sectors != null)
```

Comment:
I would change this to  @if (Model.Sectors != null || Model.Sectors.Any())

Then if its an empty list it will step over and add the dash

Reply:
I didn't change this code, so i'll leave it as is - maybe it can be revisited